File: Selenium/Project/PActivity1.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

# Set up the Firefox Driver with WebDriverManger
service = FirefoxService(GeckoDriverManager().install())

# Start the Driver
with webdriver.Firefox(service=service) as driver:
    # Open the browser to the URL
    driver.get("http://alchemy.hguy.co/crm")
    # Get the title of the website
    page_title = driver.title
    print("Title of the page is: ", page_title)
    # Make sure title matches “SuiteCRM” exactly
    assert "SuiteCRM" == page_title

File: Selenium/Project/PActivity1.java
package Project;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;


public class PActivity1 {
    public static void main(String[] args) throws InterruptedException {
        // Setup the Firefox driver(GeckoDriver)
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE, "/dev/null");
        WebDriverManager.firefoxdriver().setup();
        // Create a new instance of the Firefox driver
        WebDriver driver = new FirefoxDriver();
        driver.get("http://alchemy.hguy.co/crm");
        // Get the title of the website
        String PageTitle = driver.getTitle();
        System.out.println("Title of the page: " + PageTitle);
        // Make sure title matches “SuiteCRM” exactly.
        Assert.assertEquals("SuiteCRM",  PageTitle);
        // close the browser
        driver.quit();
    }
}
